dlp/api/DlpTest/DlpTest.cs

```diff
@@ -15,369 +15,350 @@
 using Google.Api.Gax.ResourceNames;
 using Google.Apis.Auth.OAuth2;
 using Google.Cloud.Dlp.V2;
+using GoogleCloudSamples;
 using System;
 using System.IO;
 using System.Text.RegularExpressions;
 using Xunit;
 
-namespace GoogleCloudSamples
+public class DlpTestFixture
```

Comment:
In tests we can have namespaces. We are not expecting users to need to copy/paste tests.

Reply:
Ack.